src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +43,11 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        //List에 담긴 모든 숫자 중 3보다 큰 숫자를 2배 한 후 모든 값의 합을 구한다
+        return numbers.stream()
+                .filter(number -> number > 3)
+                .mapToInt(number -> number * 2)
+                .reduce(Integer::sum)
+                .getAsInt();
```

Comment:
reduce 메서드는 `Optionalnt` 타입으로 반환하고 있어서 예기치 못한 버그가 발생할 수 있습니다.
초기값을 설정하는 reduce 메서드를 사용해보면 어떨까요?
```suggestion
        return numbers.stream()
                .filter(number -> number > 3)
                .mapToInt(number -> number * 2)
                .reduce(0, Integer::sum);
                
```